app/src/main/res/layout/activity_list.xml

```diff
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
```

Comment:
Unused namespace declaration removed that was not necessary for the code.

Reply:
Are you certain this is not needed?